client/src/App.js

```diff
@@ -22,7 +24,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList}/>
+        <Route path="/movies/:id" component={Movie}/>
```

Comment:
Compare this line with line 14 in Movie.js. The `:id` you see here is the reason why you had to write `this.props.match.params.id`. If that string had said `:banana` then you would have to write `this.props.match.params.banana`. Do you see the pattern here?